src/System.Runtime.InteropServices/System.Runtime.InteropServices.sln

```diff
@@ -26,10 +26,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{A824F4CD-935B-4496-A1B2-C3664936DA7B}.Debug|Any CPU.ActiveCfg = netcoreapp-Debug|Any CPU
-		{A824F4CD-935B-4496-A1B2-C3664936DA7B}.Debug|Any CPU.Build.0 = netcoreapp-Debug|Any CPU
-		{A824F4CD-935B-4496-A1B2-C3664936DA7B}.Release|Any CPU.ActiveCfg = netcoreapp-Release|Any CPU
-		{A824F4CD-935B-4496-A1B2-C3664936DA7B}.Release|Any CPU.Build.0 = netcoreapp-Release|Any CPU
+		{A824F4CD-935B-4496-A1B2-C3664936DA7B}.Debug|Any CPU.ActiveCfg = uapaot-Debug|Any CPU
```

Comment:
I suspect the changes to this file will get reset once you run UpdateVSConfiguration target.